README.md

```diff
@@ -255,18 +253,20 @@ if err != nil {
 }
 ```
 
+<!-- GO EXAMPLE
+}
+-->
+
 ## Known limitations (work in progress)
 
 As the library is actively developed couple of features are not there yet:
 
-- Realtime connection recovery is not implemented
-- Realtime connection failure handling is not implemented
-- ChannelsOptions and CipherParams are not supported when creating a Channel
+- Realtime connection failure handling is partially implemented
 - Realtime Ping function is not implemented
 
 ## Release process
 
-This library uses [semantic versioning](http://semver.org/). For each release, the following needs to be done:
+Starting with release 1.2, this library uses [semantic versioning](http://semver.org/). For each release, the following needs to be done:
```

Comment:
The examples use `1.1.x`, but this says semver only started with `1.2.x`, so should the examples reference `1.2.x`, or was semver already in use for `1.1.x`?

Reply:
Using 1.2.0 now as an example at df19c8b.